FTBFS against tzdata 2020a

#### Expected behaviour

Successful build.

#### Actual behaviour

```
   dh_auto_test
	make -j1 check VERBOSE=1
make[1]: Entering directory '/<<PKGBUILDDIR>>'
Making check in po
make[2]: Entering directory '/<<PKGBUILDDIR>>/po'
make[2]: Nothing to be done for 'check'.
make[2]: Leaving directory '/<<PKGBUILDDIR>>/po'
Making check in po-locations
make[2]: Entering directory '/<<PKGBUILDDIR>>/po-locations'
make[2]: Nothing to be done for 'check'.
make[2]: Leaving directory '/<<PKGBUILDDIR>>/po-locations'
Making check in libmateweather
make[2]: Entering directory '/<<PKGBUILDDIR>>/libmateweather'
make  check-am
make[3]: Entering directory '/<<PKGBUILDDIR>>/libmateweather'
make[3]: Nothing to be done for 'check-am'.
make[3]: Leaving directory '/<<PKGBUILDDIR>>/libmateweather'
make[2]: Leaving directory '/<<PKGBUILDDIR>>/libmateweather'
Making check in doc
make[2]: Entering directory '/<<PKGBUILDDIR>>/doc'
make[2]: Nothing to be done for 'check'.
make[2]: Leaving directory '/<<PKGBUILDDIR>>/doc'
Making check in data
make[2]: Entering directory '/<<PKGBUILDDIR>>/data'

vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv
xmllint --valid --noout ../data/Locations.xml.in
./check-timezones.sh ./Locations.xml.in
Invalid timezones in ./Locations.xml.in: America/Godthab
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^


make[2]: *** [Makefile:637: check] Error 1
make[2]: Leaving directory '/<<PKGBUILDDIR>>/data'
make[1]: *** [Makefile:490: check-recursive] Error 1
make[1]: Leaving directory '/<<PKGBUILDDIR>>'
dh_auto_test: error: make -j1 check VERBOSE=1 returned exit code 2
make: *** [debian/rules:10: binary] Error 25
dpkg-buildpackage: error: debian/rules binary subprocess returned exit status 2
--------------------------------------------------------------------------------
Build finished at 2020-05-22T16:41:15Z
```

#### Steps to reproduce the behaviour

Rebuild libmateweather 1.24.0 on current Debian unstable (containing tzdata 2020a-1).

#### MATE general version

1.24

#### Package version

1.24.0-1

#### Linux Distribution

Debian

#### Link to downstream report of your Distribution

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=959545

--- a/data/Locations.xml.in
+++ b/data/Locations.xml.in
@@ -6482,7 +6482,7 @@
             -->
           <name>Danmarkshavn</name>
         </timezone>
-        <timezone id="America/Godthab">
+        <timezone id="America/Nuuk">
           <!-- The primary timezone for Greenland, although sources
                seem to point towards calling the area "Western
                Greenland" rathern than just "Greenland".
@@ -6502,7 +6502,7 @@
           <name>Thule AFB</name>
         </timezone>
       </timezones>
-      <tz-hint>America/Godthab</tz-hint>
+      <tz-hint>America/Nuuk</tz-hint>
       <city>
         <!-- A city in Greenland.
              The local name in Kalaallisut is "Uummannaq".

